src/asm/rgbasm.5

```diff
@@ -260,19 +260,38 @@ PRINTT "The answer to {TOPIC} is {ANSWER}\[rs]n"
 .Pp
 Symbol interpolations can be nested, too!
 .Pp
-It's possible to change the way numeric symbols are converted by specifying a print type like so:
-.Ql {d:symbol} .
+It's possible to change the way symbols are converted by specifying a print format like so:
+.Ql {fmt:symbol} .
+The
+.Ql fmt
+specifier consists of parts
+.Ql <sign><prefix><align><pad><width><frac><type> .
+These parts are:
+.Bl -column "<prefix>"
+.It Sy Part Ta Sy Meaning
+.It Ql <sign> Ta May be Qo + Qc or Ql \  . If specified, prints this character in front of non-negative numbers.
+.It Ql <prefix> Ta May be Ql # . If specified, prints the appropriate prefix for numbers, Qo $ Qc , Qo & Qc , or Ql % .
```

Comment:
I should ask @bentley how much we can rely on a period ending a `Ql`... Besides, since it's not at EOL, it's not treated as an end-of-sentence, is it?

Reply:
I've checked the formatting with Cygwin's `man`, but am not 100% sure about it, since even the 0.4.2 `rgbasm(5)` page has some odd single quotes in it that don't appear in the web render.

Reply:
Them not appearing is normal, I believe. Again, I'd need bentley's expertise to be sure.

Reply:
I installed mandoc 1.14.5 in Cygwin and got both the [`mandoc` HTML output](https://i.imgur.com/IuKWm90.png) and the [console `man` view](https://i.imgur.com/dOTaMaq.png) of `rgbasm.5` to look correct.